### ISSUE TITLE
tslint --fix produces syntax errors

### ISSUE THREAD

**Issue by angelo-mollame-2** [palantir/tslint#4857]
### Bug Report

- __TSLint version__: 5.16.0
- __TypeScript version__: 3.5.1
- __Running TSLint via__: CLI

#### Reproduction
This seems to be the same issue as #2862. I am seeing the issue with a custom TSLint rule whose fixer should do nothing but add a blank line over some statement nodes.

Repro steps:
- Download [tslint-repro.zip](https://github.com/palantir/tslint/files/3615290/tslint-repro.zip)
- Run these commands:
```
npm install
npm install --dev
node_modules\.bin\tsc tslint-rules\statementPaddingRule.ts
node_modules\.bin\tslint -p tsconfig.json -c tslint.json --fix
```

#### TypeScript code being linted
This is **test1.ts** in the ZIP file.

```ts
export function foo(): any {
    return () => {
        doSomething();
        const a: string = getSomething();
        if (getSomething()) {
            if (getSomething()) {
                const bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb: any = getSomething(
                    "foo"
                );
                if (getSomething()) {
                    const c: any = getSomething();
                    if (getSomething()) {
                        doSomething();
                    }
                }
            }
        }
    };
}

function doSomething(): void {

}

function getSomething(foo?: any): any {

}
```

#### Actual behavior

```
export function foo(): any {
    return () => {
        doSomething();
        const a: string = getSomething();

        if (getSomething()) {
            if (getSomething()) {
                const bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb: any = getSomething(
                    'foo',
                

                if (getSomething()) {
                    const c: any = getSomething(

                    if (getSomething()) {
                        doSomething();
                    } }
                } }
            }
        }
    };
}

function doSomething(): void {

}

function getSomething(foo?: any): any {

}
```

#### Expected behavior
The fixer should just add blank lines above some statements, but TSLint doesn't seem to apply the fixes sequentially. It seems like it tries to apply them all at once, and fixes further down probably operate on the old file content, even though previous fixes had shifted it, which messes up the final result.

I checked the repro steps in #2862 (the content is in **test2.ts**, and the rules that #2862 used are included in my tslint.json), and **test2.ts** is actually fixed correctly, as part of the same `tslint --fix` command.

**Comment by adidahiya** [comment]
I think this is essentially the same as #2556, which, unfortunately, is unlikely to get resolved since TSLint is being deprecated soon (#4534).

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
